tests/build.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path')
+const test = require('ava')
+const execa = require('execa')
+const yup = require('yup')
+
+test('builds a JSON object with a valid schema', async t => {
+  const schema = yup.array(
+    yup.object({
+      name: yup.string().required(),
+      // `array().required()` doesn't work for `keywords` because
+      // `array().required()` treats `[]` as a missing value.
+      // `mixed().required()` ensures that the property is defined,
+      // but doesn't ensure that it's an array.
+      keywords: yup.array().ensure(),
+      width: yup.number().required(),
+      height: yup.number().required(),
+      path: yup.string().required()
+    })
+  )
+
+  const {stdout} = await execa(path.resolve(__dirname, '../script/build.js'), ['--input=../icons/**/*.svg'], {
+    cwd: __dirname
+  })
+
+  const icons = JSON.parse(stdout)
+
+  t.true(schema.isValidSync(Object.values(icons)))
+})
```

Comment:
If anyone knows of a better way to test the output of the `build` script, I'd love to hear it 😸